packages/mc-http-server/server.js

```diff
@@ -3,6 +3,8 @@ const fs = require('fs');
 const path = require('path');
 const http = require('http');
 const morgan = require('morgan');
+const hidePoweredBy = require('hide-powered-by');
```

Comment:
Just so we use the same package across all services. It's not really needed but I somehow value consistency here.